docs/tutorials/mongoid-transactions.txt

```diff
@@ -0,0 +1,115 @@
+============
+Transactions
+============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Version 4.0 of the MongoDB server introduces
+`multi-document transactions <https://docs.mongodb.com/manual/core/transactions/>`_.
+(Updates to multiple fields within a single document are atomic in all
+versions of MongoDB). Mongoid versions 6.4 and higher support transactions.
```

Comment:
I think specifying that driver 2.6.0 or higher is required is important here, because using driver 2.6 with older mongoids will make it possible to use transactions. I wouldn't give a mongoid version here at all unless you want to figure out which version the session support was added in, and also add that to the sessions page because it currently doesn't say.